include/openrave/collisionchecker.h

```diff
@@ -88,6 +88,7 @@ class OPENRAVE_API CollisionReport
     //@{
     KinBody::LinkConstPtr plink1, plink2; ///< the colliding links if a collision involves a bodies. Collisions do not always occur with 2 bodies like ray collisions, so these fields can be empty.
     KinBody::GeometryConstPtr pgeom1, pgeom2; ///< the specified geometries hit for the given links
```

Comment:
Could you add a comment saying these might be null if Collision is coming from geometry info?